components/03-modules/slider/slider.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="slider {{ class }}" {{{ attributes }}}>
+    <div class="slider__handler">
+        <div class="slider__slides">
+            {{#each products }}
+                <div class="slider__item">
+                    <a href="{{ this.link }}" class="slider__link">
```

Comment:
Now I see that we might use some `aria-label` statements :D ?

Reply:
added